docs/timeseries/analysis.rst

```diff
@@ -98,20 +111,28 @@ Sorting time series in-place can be done using the
     2016-03-22T12:30:34.000     4.0
     2016-03-22T12:30:37.000     5.0
 
+.. EXAMPLE END
+
 Resampling
 ==========
 
 We provide a :func:`~astropy.timeseries.aggregate_downsample` function
 that can be used to bin values from a time series into bins of equal time, using
 a custom function (mean, median, etc.). This operation returns a
-|BinnedTimeSeries|. Note that this is a simple function in the sense that it
-does not for example know how to treat columns with uncertainties differently
+|BinnedTimeSeries|. Note that this is a minimal function in the sense that it
```

Comment:
What about concise? Or maybe the use of simple here is benign since it's referring to the simplicity of the function, not the assumed simplicity of what the user needs to do.

Reply:
simple or basic sounds the best to me as it's indeed about the scope of functionality not about the usage.